cmake/Reprojucer.cmake

```diff
@@ -183,6 +184,16 @@ function(jucer_project_settings)
       elseif(tag STREQUAL "PREPROCESSOR_DEFINITIONS")
         string(REPLACE "\n" ";" value "${value}")
 
+      elseif(tag STREQUAL "HEADER_SEARCH_PATHS")
+        string(REPLACE "\\" "/" value "${value}")
+        string(REPLACE "\n" ";" value "${value}")
```

Comment:
Does this work correctly with all types of line endings, if someone writes a `CMakeLists.txt` directly without using the converter?

Reply:
CMake takes care of treating `\r\n` like `\n`, so we don't have to think about it.